imap_processing/utils.py

```diff
@@ -64,8 +69,20 @@ def convert_raw_to_eu(dataset: xr.Dataset, conversion_table_path, packet_name):
         Path to engineering unit conversion table.
         Eg:
         f"{imap_module_directory}/swe/l1b/engineering_unit_convert_table.csv"
+        Engineering unit conversion table must be a csv file with required
+        informational columns: ('packetName', 'mnemonic', 'convertAs') and
+        conversion columns named 'c0', 'c1', 'c2', etc. Conversion columns
+        specify the array of polynomial coefficients used for the conversion.
+        Comment lines are allowed in the csv file specified by starting with
+        the '#' character.
     packet_name: str
         Packet name
+    read_csv_kwargs: dict
```

Comment:
Numpydoc should have a space between the parameter and type.
```suggestion
    read_csv_kwargs : dict
```